style.css

```diff
@@ -422,6 +501,238 @@ form textarea:focus {
 }
+
+div {
+  display: block;
+}
+
+#section-contact p {
+  font-family: "inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: #42526e;
+  margin-top: 1em;
+  margin-bottom: 3em;
+}
+
+#textarea {
+  background-image: url(./imges/Icon\ -\ Resizer\ Handle.svg);
+  background-repeat: no-repeat;
+  background-position: left 325px, 200px 385px, right 280px;
+  background-size: 130px;
+}
+
+#mobile-menu {
+  height: 85vh;
+  border-bottom-right-radius: 2em;
+  border-bottom-left-radius: 2em;
+  margin-bottom: 2em;
+  mix-blend-mode: normal;
+  box-shadow: 0 1em 1em rgba(37, 47, 137, 0.08);
+}
+
+#mobile-menu > ul {
+  clear: right;
+  padding-left: 0;
+}
+
+.menu-link {
+  display: block;
+  text-decoration: none;
+  margin: 1em 1em;
+  color: #3a4a42;
+  font-size: x-large;
+  font-weight: 700;
+}
+
+.hidden {
+  display: none;
+}
+
+.social-media-icons li:hover {
+  transform: scale(1.3);
+}
+
+@media (min-width: 768px) {
+  /* Standard CSS rules */
+  html {
+    box-sizing: border-box;
+  }
+
+  header > a,
+  header > img {
+    display: none;
+  }
+
+  header {
+    padding: 0;
+  }
+
+  #headline {
+    background-image: url("./imges/Desktop/Header-llustration.png");
+    background-size: 106% 16%;
+  }
+
+  .my-placeholder {
+    grid-template-columns: repeat(3, 1fr);
+    height: 372px;
+    width: 229px;
+  }
+
+  nav {
+    margin-top: -1em;
+  }
+
+  .middle {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  nav a {
+    text-decoration: none;
+    color: #172b4d;
+    font-size: large;
+    font-weight: 550;
+  }
+
+  nav a:focus {
+    color: #008552;
+  }
+
+  .only-desktop {
+    display: block;
+  }
+
+  .tittleContents > h2 {
+    font-size: 275%;
+  }
+
+  .tittleContents > h1 {
+    font-size: 275%;
+  }
+
+  .primary > h1 {
+    font-size: 275%;
+  }
+
+  .img1-aboutme > p {
+    font-size: 150px;
+  }
+
+  .grid {
+    width: 95%;
+    margin: 2em;
+    grid-template-columns: repeat(3, minmax(300px, 1fr));
+    gap: 5px;
+  }
+
+  #aboutme {
+    grid-template-columns: repeat(3, 1fr);
+    padding-bottom: 11px;
+    margin-bottom: 5%;
+    border: solid 1px #d0d9d4;
+    background-color: #ebf0ee;
+    border-radius: 10px 10px 10px 10px;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    height: 100%;
+    width: 100%;
+  }
+
+  .img1-aboutme {
+    background-image: url("./imges/Desktop/about-me-illustration1.png");
+    background-size: 100px;
+    background-position-y: -15px;
+  }
+
+  .img1-aboutme + h2 {
+    padding: 0;
+  }
+
+  .supporting-text > p {
+    font-size: 150%;
+  }
+
+  .skills-card {
+    margin: 22px;
+    display: grid;
+    gap: 24px;
+    justify-items: center;
+  }
+
+  #about-me > p {
+    text-align: center;
+    padding: 0 5em;
+    margin: 0;
+  }
+
+  .img2-aboutme {
+    grid-template-columns: repeat(3, 1fr);
+  }
```

Comment:
- [ ] Please kindly make sure your `About` section has all the backgrounds images 😁 
- [ ] Also kindly change the font of the paragraph, it is not the same as in the Figma design 😉 
- [ ] Make sure your skills-card are the same as in the Figma design 😁 

Your design

<img width="822" alt="Screen Shot 2022-11-12 at 00 28 32" src="https://user-images.githubusercontent.com/74501523/201440738-8dfa23ee-857f-4afd-92fb-022724a35325.png">

Figma design

<img width="722" alt="Screen Shot 2022-11-12 at 00 28 12" src="https://user-images.githubusercontent.com/74501523/201440760-00a5b114-03b4-4d0e-a52c-d7e456d75262.png">

Reply:
- [ ] Please stick to the Figma template on the technologies list and align the text and use the provided icons.

Figma template

<img width="697" alt="Screen Shot 2022-11-12 at 8 12 04 PM" src="https://user-images.githubusercontent.com/47673311/201498602-e2882d42-cdc2-4ebf-a66d-574fa4d33a11.png">

Your design

<img width="1280" alt="Screen Shot 2022-11-12 at 8 12 30 PM" src="https://user-images.githubusercontent.com/47673311/201498636-a12f9897-572b-47dd-8122-002833790450.png">

Icons

<img width="1278" alt="Screen Shot 2022-11-12 at 8 27 38 PM" src="https://user-images.githubusercontent.com/47673311/201498652-08a260e2-e44a-42ab-8c3c-2ed0773d7238.png">